extensions/fake-financial-data/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "fake-financial-data",
+  "title": "Fake Financial Data",
+  "description": "Generate fake financial data",
+  "icon": "command-icon.png",
+  "author": "andre-rd-rodrigues",
+  "categories": [
+    "Finance"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "ffd-commands",
+      "title": "Generate Financial Data",
+      "subtitle": "Generate Financial data",
```

Comment:
Hey @andre-rd-rodrigues 👋,

in this case I don't think a subtitle is needed, the title is self explanatory.

> Don't use a subtitle if it doesn't add context. Usually, this is the case with single command extensions.
 _Raycast Docs_

Check out the docs regarding this: https://developers.raycast.com/basics/prepare-an-extension-for-store